src/vs/workbench/contrib/chat/browser/chatAccessibilityProvider.ts

```diff
@@ -69,13 +82,13 @@ export class ChatAccessibilityProvider implements IListAccessibilityProvider<Cha
 		const codeBlockCount = marked.lexer(element.response.toString()).filter(token => token.type === 'code')?.length ?? 0;
 		switch (codeBlockCount) {
 			case 0:
-				label = accessibleViewHint ? localize('noCodeBlocksHint', "{0}{1}{2} {3}", toolInvocationHint, fileTreeCountHint, element.response.toString(), accessibleViewHint) : localize('noCodeBlocks', "{0} {1}", fileTreeCountHint, element.response.toString());
+				label = accessibleViewHint ? localize('noCodeBlocksHint', "{0}{1}{2}{3} {4}", toolInvocationHint, fileTreeCountHint, tableCountHint, element.response.toString(), accessibleViewHint) : localize('noCodeBlocks', "{0} {1}", fileTreeCountHint, element.response.toString());
```

Comment:
You probably don't need to localize these strings that just have placeholders and no content :)

Reply:
I do wonder if some spaces might get lost from the other localized strings though. I wouldn't count on trailing spaces to survive localization